### PR TITLE
feat(v2 command): Update the API path to /device/name/{name}/{command}

### LINKED ISSUE #459: [V2 command]  Update the API path to /device/name/{name}/{command}
Per discussion from @jpwhitemn, @lenny-intel, and @cloudxxx8 below, Core Command V2 API GET/PUT /device/name/{deviceName}/command/{commandName} should be changed to the same path used by device-service-sdk, which is GET/PUT /device/name/{name}/{command}

### PR DESCRIPTION
**Pull request by judehung**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/.github/Contributing.md.

## Issue Number:
fix #459 

## What is the new behavior?
Per discussion from @jpwhitemn, @lenny-intel, and @cloudxxx8, Core Command V2 API GET/PUT /device/name/{deviceName}/command/{commandName} should be changed to the same path used by device-service-sdk, which is GET/PUT /device/name/{name}/{command}

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by judehung** [comment]
@hahattan for your awareness, please check if this PR will make any impact to device-sdk-go

File: v2/constants.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ const (
 	ApiDeviceByProfileNameRoute   = ApiDeviceRoute + "/" + Profile + "/" + Name + "/{" + Name + "}"
 	ApiDeviceByServiceIdRoute     = ApiDeviceRoute + "/" + Service + "/" + Id + "/{" + Id + "}"
 	ApiDeviceByServiceNameRoute   = ApiDeviceRoute + "/" + Service + "/" + Name + "/{" + Name + "}"
-	ApiDeviceNameCommandNameRoute = ApiDeviceRoute + "/" + Name + "/{" + DeviceName + "}" + "/" + Command + "/{" + CommandName + "}"
```
Comment (cloudxxx8) [inline v2/constants.go:55]: please also remove the CommandName constant

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: v2/constants.go (hunk 1 above)
Comment (judehung) [reply v2/constants.go:55]: https://github.com/edgexfoundry/go-mod-core-contracts/pull/460/commits/673266bae183075f6fd13f200aa979784cf454e1 removes commandName constant.

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**